lib/CL/pocl_llvm_build.cc

```diff
@@ -355,14 +355,14 @@ int pocl_llvm_build_program(cl_program program,
 
   POCL_MSG_PRINT_LLVM("all build options: %s\n", ss.str().c_str());
 
-  char replace_me = 0;
+  char WSReplacementChar = 0;
 
   char *temp_options = (char *) malloc (ss.str().length() + 1);
```

Comment:
This also needs renaming. Please check: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly    Thanks!